### ISSUE TITLE
[Feature Request] Multi Output

### ISSUE THREAD

**Issue by santient** [keras-rl/keras-rl#228]
I saw that multi-input is possible using a MultiInputProcessor as well as a pull request adding multi input functionality.
Would it be possible to implement multi-output as well?
For example, with this feature, it would be possible for the agent model to output a mix of discrete and continuous action spaces, with, for example, the discrete ones representing "what should I modify in the env" and the continuous ones representing "by how much".
I'm not sure which RL algorithm would be implemented in this case, as there are different ones for discrete vs continuous action spaces.
This would be very useful especially if it allowed dict outputs from the model.

**Comment by gvgramazio** [comment]
@santient 

tl;dr: This should be handled by the processor instead that by the agent.

State a case when this could be helpful. If you simply want the discrete action to say "what" and continuous to say "how much" you could simply use continuous action space for all these actions and then, in the processor, convert some of them to discrete (in this case a simple boolean one). Also, if some of these boolean actions activate/deactivate some of the continuous actions and it's not handled in the environment you could handle by yourself in the processor by setting to zero these continuous actions or something similar.
